### PR TITLE
Fix gradle command not found error for Python ValidatesContainer Dataflow ARM

### PR DESCRIPTION
**Pull request by celeste-zeng**
- Updated yml file Python ValidatesContainer Dataflow ARM to fix gradle command not found error
- Add comments and else if condition for CPU architecture argument for the validatescontainer script.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by celeste-zeng** [comment]
Run Python Validates Container Dataflow ARM 3.10

**Comment by celeste-zeng** [comment]
Run Python ValidatesContainer Dataflow ARM 3.11

**Comment by tvalentyn** [comment]
Run Python ValidatesContainer Dataflow ARM

File: .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml [hunk 1]
```diff
@@ -49,15 +49,16 @@ concurrency:
 
 jobs:
   beam_Python_ValidatesContainer_Dataflow_ARM:
-    name: beam_Python_ValidatesContainer_Dataflow_ARM
+    name: beam_Python_ValidatesContainer_Dataflow_ARM (Run Python ValidatesContainer Dataflow ARM 3.x)
```
Comment (tvalentyn) [inline .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml:52]: Could we remove 3.x from the name and trigger commands?

File: .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml [hunk 2]
```diff
@@ -107,17 +108,18 @@ jobs:
...
           arguments: |
             -PpythonVersion=${{ matrix.python_version }} \
             -Pcontainer-architecture-list=arm64,amd64 \
             -Pdocker-repository=us.gcr.io/apache-beam-testing/github-actions \
             -Pdocker-tag=${{ env.MULTIARCH_TAG }} \
-            -Ppush-containers
-          env:
-            MULTIARCH_TAG: ${{ env.MULTIARCH_TAG }}
-            USER: github-actions
+            -Ppush-containers \
```
Comment (tvalentyn) [inline .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml:119]: is the last line-joining intentional?

**Review by tvalentyn — APPROVED** [review]

File: .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml (hunk 2 above)
Comment (celeste-zeng) [reply .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml:119]: I just followed what was done here: https://github.com/apache/beam/blob/afa4371b1ddaf017c817d9a833d3a6d04310236c/.github/workflows/beam_PreCommit_Python_Integration.yml#L116

File: .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml (hunk 1 above)
Comment (celeste-zeng) [reply .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml:52]: Followed what was done here: https://github.com/apache/beam/blob/afa4371b1ddaf017c817d9a833d3a6d04310236c/.github/workflows/beam_PreCommit_Python_Integration.yml#L54

Comment (tvalentyn) [reply .github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml:52]: looks like we do need to specify matrix element. then, it's intentional. nvm.

**Pull request merged**